python/ray/actor.py

```diff
@@ -9,6 +9,7 @@
 import ray.worker
 from ray.util.placement_group import (
     PlacementGroup, check_placement_group_index, get_current_placement_group)
+from ray.util.collective_utils import collective_to_envs
```

Comment:
If you are adding this to `actor.py`, can you be sure to add a test in the core library actor tests?

Reply:
Thx for pointing out. Added under test_actor.py.